x/language.rb

```diff
@@ -0,0 +1,18 @@
+module X
+  class Language
+    def self.of(id)
+      tracks[id.to_s]
+    end
+
+    def self.tracks
+      @@tracks ||= fetch_tracks.inject({}) do |hash, track|
+        hash[track.id] = track.language
+        hash
```

Comment:
Inject doesn't seem like the right choice of enumerable if you need an extra line.